src/crush/CrushCompiler.cc

```diff
@@ -203,14 +203,16 @@ int CrushCompiler::decompile(ostream &out)
   if (crush.get_allowed_bucket_algs() != CRUSH_LEGACY_ALLOWED_BUCKET_ALGS)
     out << "tunable allowed_bucket_algs " << crush.get_allowed_bucket_algs()
 	<< "\n";
+  if (crush.get_reweight_items() != 0)
+    out << "tunable reweight_items " << crush.get_reweight_items() << "\n";
```

Comment:
I wonder if we can come up with a more descriptive name.. 'reweight_multi_pick" or something?